cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -1,59 +1,26 @@
-/*
-Copyright 2016 The Kubernetes Authors.
```

Comment:
We still need a K8s Author copyright notice.

Reply:
Added back. Thanks

Reply:
We need to lint for this in this repo. I'll track that separately.